packages/blade/src/components/Typography/Text/Text.tsx

```diff
@@ -12,19 +12,19 @@ type TextCommonProps = {
   contrast?: ColorContrastTypes;
   truncateAfterLines?: number;
   children: React.ReactNode;
+  weight?: keyof Theme['typography']['fonts']['weight'];
+  _color?: BaseTextProps['color'];
```

Comment:
TODO: Remove these changes if #837 merges before this.

Reply:
Not needed anymore since this PR will be merged first.

Reply:
🥇